specification/dsdl/serializable_types.tex

```diff
@@ -0,0 +1,285 @@
+\section{Serializable types}\label{sec:dsdl_serializable_types}
+
+Values of the serializable type category can be exchanged between nodes over the UAVCAN network.
+This is opposed to the expression types (section \ref{sec:dsdl_expression_types}),
+instances of which can only exist while DSDL definitions are being evaluated.
+The data serialization rules are defined in the section \ref{sec:dsdl_data_serialization}.
+
+\subsection{Void types}
+
+Void types are used for padding purposes.
+As will be explained in later sections, it is desirable to align field attributes at byte boundaries;
+void types can be used to facilitate that.
+
+Void-typed field attributes are set to zero when an object is serialized and ignored when it is deserialized.
+Void types can be used to reserve space in data type definitions for possible use in later versions of the data type.
+
+The DSDL name pattern for void types is as follows: ``\verb|void[1-9]\d*|'',
+where the trailing integer represents its width, in bits,
+ranging from 1 to 64, inclusive.
+
+Void types can be referred to directly by their name from any namespace.
```

Comment:
Huh. I thought void types were anonymous. We can actually read these values?

Reply:
This is about types, not their values. This merely means that void types are not contained within some particular namespace, they are built-in, like `void` in C++.